woocommerce/woocommerce-android
# [Tablet Orders] Add a drawer to the order creation screen

Closes: #10424


### Description

Adds a view that looks like bottoms sheet to the order creation/editing. Behind feature flag

The UI is not implemented in this PR, it's just a skeleton for the future changes

### Testing instructions

* Try debug/release builds
* Notice that the drawer is not visible on the release buid

### Images/gif

<img width="282" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/73e3085f-b81e-4cc5-ba81-bd7198ecbdff">
